contributing/README.md

```diff
@@ -10,13 +10,38 @@ us on [Slack]. Please also take a look at our [code of conduct], which details
 how contributors are expected to conduct themselves as part of the Crossplane
 community.
 
+## Establishing a Development Environment
+
+> The Crossplane project consists of several repositories under the crossplane
+> and crossplane-contrib GitHub organisations. We're experimenting with
+> [Earthly] in this repository (crossplane) and crossplane-runtime. Most other
+> repositories use a `Makefile`. To establish a development environment for a
+> repository with a `Makefile`, try running `make && make help`.
+
+Crossplane is written in [Go]. You don't need to have Go installed to contribute
+code to Crossplane but it helps to use an editor that understands Go.
+
+To setup a Crossplane development environment:
+
+1. Fork and clone this repository.
+1. Install [Docker][get-docker] and [Earthly][get-earthly].
+
+Use the `earthly` command to build and test Crossplane. Run `earthly doc` to see
+available build targets.
+
+Useful targets include:
+
+* `earthly +reviewable` - Run code generators, linters, and unit tests.
+* `earthly +e2e` - Run end-to-end tests.
```

Comment:
I tried this command and got this error:
```
                +e2e | --> FROM +base
Error: Earthfile:47:2 apply ARG: value not supplied for required ARG: FLAGS
in		+e2e --FLAGS=
```

Since `--FLAGS` appears to be required, is it worth updating this example so it's clear to the new contributor what they should be passing for this value?

Reply:
I pushed a commit that gives `FLAGS` some sane defaults so `earthly -P +e2e` will work.

I also updated the e2e test documentation to talk about Earthly rather than Make.

Reply:
nice yes, looks like with your latest commits `earthly -P +e2e` is running the e2e tests for me.